### ISSUE TITLE
JSON Decode Errors

### ISSUE THREAD

**Issue by lxndrblz** [lxndrblz/forensicsim#75]
Error Log:

```text
buddylist skypexspaces-contacts-<GUID_1> (Records: 5)
replychains skypexspaces-<GUID_1> (Records: 0)
conversations skypexspaces-<GUID_1> (Records: 0)
people skypexspaces-<GUID_1> (Records: 1924)
people Teams:substrate-suggestions-manager:<GUID_2> (Records: 0)
replychains Teams:replychain-manager:<GUID_3> (Records: 13225)
conversations Teams:conversation-manager:<GUID_3> (Records: 1358)
Traceback (most recent call last):
  File "main.py", line 65, in <module>
  File "click\core.py", line 1157, in __call__
  File "click\core.py", line 1078, in main
  File "click\core.py", line 1434, in invoke
  File "click\core.py", line 783, in invoke
  File "main.py", line 61, in process_cmd
  File "forensicsim\parser.py", line 373, in process_db
  File "forensicsim\parser.py", line 355, in parse_records
  File "forensicsim\parser.py", line 245, in _parse_conversations
  File "dataclasses_json\api.py", line 70, in from_dict
  File "dataclasses_json\core.py", line 206, in _decode_dataclass
  File "forensicsim\parser.py", line 36, in decode_dict
  File "json\__init__.py", line 359, in loads
  File "json\decoder.py", line 337, in decode
  File "json\decoder.py", line 355, in raw_decode
json.decoder.JSONDecodeError: Expecting value: line 1 column 2 (char 1)
[15784] Failed to execute script 'main' due to unhandled exception!
```

**Comment by kyleEeeEEeeee** [comment]
I am also getting this issue

**Comment by minit01** [comment]
Same issue on my side - is there anything we can do to support you in getting the fix candidate #76 into an official release soon ? :)
